src/documentation/programming-guide.md

```diff
@@ -95,6 +95,8 @@ public static void main(String[] args) {
 ```py
 # Will parse the arguments passed into the application and construct a PipelineOptions object.
 # Note that --help will print registered options.
+import sys
```

Comment:
There is no need to add this, to the snippet. `PipelineOptions()` does this automatically.